recipes/fpng/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, save, load
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class FpngConan(ConanFile):
+    name = "fpng"
+    description = "Super fast C++ .PNG writer/reader"
+    license = "unlicense",
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/richgel999/fpng"
+    topics = ("png", "writer", "reader")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_sse": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_sse": False,
```

Comment:
```suggestion
        "with_sse": True,
```
According to https://store.steampowered.com/hwsurvey/ (other settings), SSE4.1 support is at 99.66% in consumer-grade hardware. I would enable it by default.